tests/framework/Optimizers/GradientBasedOptimizers/test_spsaWConstraint.xml

```diff
@@ -61,30 +61,30 @@
   <Optimizers>
     <SPSA name="opt_smp">
       <initialization>
-        <limit>1000</limit>
+        <limit>10000</limit>
```

Comment:
How long does the 10k samples take?  Or is this just a high limit so we hit IterationLimit instead?